operator/internal/controller/redpanda/resource_controller.go

```diff
@@ -22,12 +22,11 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 
 	redpandav1alpha2 "github.com/redpanda-data/redpanda-operator/operator/api/redpanda/v1alpha2"
+	"github.com/redpanda-data/redpanda-operator/operator/internal/lifecycle"
 	internalclient "github.com/redpanda-data/redpanda-operator/operator/pkg/client"
 	"github.com/redpanda-data/redpanda-operator/pkg/otelutil/log"
 )
 
-const fieldOwner client.FieldOwner = "redpanda-operator"
```

Comment:
I also discovered, via an upgrade test, that this value is different from that in the lifecycle client.